src/pypi_simple/client.py

```diff
@@ -383,7 +383,7 @@ def get_package_metadata(
         r.raise_for_status()
         digester.update(r.content)
         digester.finalize()
-        return r.text
+        return r.content.decode("utf-8")
```

Comment:
```suggestion
        return r.content.decode("utf-8", "surrogateescape")
```

Because we don't want to crash on the inevitable case where someone somehow created a wheel with non-UTF-8 metadata.